pkgs/development/python-modules/aws-lambda-builders/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "aws-lambda-builders";
+  version = "0.2.1";
+
+  src = fetchPypi {
```

Comment:
I noticed that there's no tests distributed on `pypi`.

Can you use `fetchFromGitHub` to get an archive that has them and enable them?

Reply:
@worldofpeace I've enabled the tests but it looks like tests are trying to write files.

Is that allowed in the sandbox?

Reply:
The build cannot write files outside of the build prefix.

Can you paste the errors in a gist?

Reply:
Here it is https://gist.github.com/dhl/bdc933473f1856015e550f14646936c0

Reply:
@worldofpeace managed to fix the tests by adding the package bin path in `preCheck` phase:

```
preCheck = "export PATH=$out/bin:$PATH";
```